CHANGELOG.md

```diff
@@ -4,23 +4,61 @@ This document outlines major changes between releases.
 
 ## [Unreleased]
 
+## [0.24.0] - 2022-09-14
+
 ### Added
-- Exposure of pool metrics (#615)
+
```

Comment:
I'm not sure we need this empty line after headers. The previous releases' changelogs don't have this line.